docs/docs/examples/mysql.md

```diff
@@ -0,0 +1,251 @@
+Contrast is designed such that, by verifying the Coordinator, the user transitively verifies those systems the Coordinator has already verified or will verify in the future.
+Successful verification of the Coordinator means that the user can be sure that the given manifest will be enforced.
+
+### Verifying the Coordinator
+
+A user can verify the Contrast deployment using the verify
+command:
+
+```sh
+contrast verify -c "${coordinator}:1313" -m manifest.json
+```
+
+The CLI will verify the Coordinator via remote attestation using the reference values from a given manifest. This manifest needs
+to be communicated out of band to everyone wanting to verify the deployment, as the `verify` command checks
+if the currently active manifest at the Coordinator matches the manifest given to the CLI. If the command succeeds,
+the Coordinator deployment was successfully verified to be running in the expected Confidential
+Computing environment with the expected code version. The Coordinator will then return its
+configuration over the established TLS channel. The CLI will store this information, namely the root
+certificate of the mesh (`mesh-ca.pem`) and the history of manifests, into the `verify/` directory.
+In addition, the policies referenced in the manifest history are also written into the same directory.
+
+:::warning
+On bare metal, the [coordinator policy hash](components/policies.md#platform-differences) must be overwritten using `--coordinator-policy-hash`.
+:::
+
+### Auditing the manifest history and artifacts
+
+In the next step, the Coordinator configuration that was written by the `verify` command needs to be audited.
+A user of the application should inspect the manifest and the referenced policies. They could delegate
+this task to an entity they trust.
+
+### Connecting to the application
+
+To access the MySQL database from the command line, expose the service on a public IP address via a LoadBalancer service:
+
+```sh
+mysqlIP=$(kubectl get svc mysql-svc -o=jsonpath='{.status.loadBalancer.ingress[0].ip}')
+echo "MySQL is available at the IP  $mysqlIP."
+```
+
+Using the `mysql` command line interface, you can connect to the database as the
+root user with the password specified in the deployment YAML (`password`).
+
+```sh
+mysql -h $mysqlIP -u root -p
+```
+
+:::warning
+Avoid including the MySQL root user password directly in the Kubernetes
+configuraiton files to prevent exposing sensitive credentials. Use [Kubernetes
+Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) in
+production to securely manage and reference the password instead.
+:::
+
+## Updating the deployment
+
+Because the workload secret is bound to the workload itself and not to an
+individual pod, once the pod restarts, the `cryptsetup` init container can
+deterministically generate the same key again and open the already partitioned
+LUKS device.
+
+For example, after making changes to the deployment files, the runtime policies
```

Comment:
The advice here is not correct. If you change the deployment, you're likely to get a different policy hash, resulting in a different workload secret. This is where the [`WorkloadSecretID`](https://github.com/edgelesssys/contrast/blob/55cb03624be5271088b145fbc4a547cad096afbf/internal/manifest/manifest.go#L36) comes in, which may be, well, underdocumented right now.

Reply:
Sorry, I was mistaken about the workload secret seed. Maybe change 

> the workload secret is bound to the workload itself

to something like

> the workload secret is derived from the resource name

and link to the architecture page for more details. Those details are missing right now, but should be added asap.